java/ql/test/experimental/query-tests/security/CWE-918/RequestForgery.expected

```diff
@@ -0,0 +1,84 @@
+edges
```

Comment:
Is this an accidental commit? It looks unrelated to this PR.

Reply:
Yes, it's a mistake from a merge in which this should have been removed. Thanks for spotting this! Fixed in 3656580